spec/support/features/session_helper.rb

```diff
@@ -162,7 +162,7 @@ def acknowledge_and_confirm_recovery_code
       click_on button_text, class: 'recovery-code-continue'
 
       code_words.size.times do |index|
-        fill_in "recovery-#{index}", with: code_words[index].downcase
+        fill_in "recovery-#{index}", with: code_words[index].downcase + 'xxxx'
```

Comment:
perhaps add a string with `'extra_chars_that_are_ignored'` rather than `'xxxx'` so the reasoning here is more obvious?